### ISSUE TITLE
Fuelwood Heater

### ISSUE THREAD

**Issue by Pyromanc3r** [mekanism/Mekanism#7217]
Does anyone explain to me why existing this thing? Either I do not understand something, or it works not right. 
Each Fuelwood Heater adds only 8 mB/t steam generation to boiler. I have 30 Fuelwood Heaters connected to the Boiler Casing (it is exactly Boiler Casing not Glass). 
All this array creates only 240 mB/t steam. But Industrial Turbine produces only 136 FE/t. This is ridiculous. 1 Wind Generator can produce even more. 
Looks like it is more profitable to burn coal in other generators for direct energy than to get heating from FH and convert it into steam->turbine.

Forge:36.1.23
Mekanism:10.0.21.448
MekanismGenerators:10.0.21.448
![2021-05-29_07 46 52](https://user-images.githubusercontent.com/17619385/120059025-730f4380-c057-11eb-86a2-f84001da1691.png)
![2021-05-29_07 48 46](https://user-images.githubusercontent.com/17619385/120059028-74407080-c057-11eb-9e81-8fdf1f5b737b.png)
![2021-05-29_07 49 27](https://user-images.githubusercontent.com/17619385/120059031-74d90700-c057-11eb-9039-874788fbef4f.png)
![2021-05-29_08 04 01](https://user-images.githubusercontent.com/17619385/120059032-75719d80-c057-11eb-9b71-899320ac6ee5.png)
![2021-05-29_08 04 33](https://user-images.githubusercontent.com/17619385/120059034-760a3400-c057-11eb-8503-9a9aeef3fd6f.png)

**Comment by Robball14** [comment]
From what I see, it's a problem of the boiler. 
Try Making the boiler higher, leaving a block of space between the pressure dispenser and the roof

**Comment by zaze06** [comment]
i don't know what your trying to do but the heater works whit the heat generator to generate power

**Comment by Pyromanc3r** [comment]
>  Try Making the boiler higher, leaving a block of space between the pressure dispenser and the roof

Nope. It did not help:( I added an even higher layer of the boiler + another 5 Fuelwood Heaters what gave in the amount of another 40 mB/t steam.
Fuelwood has a limit to 8 mB/t steam. That is, from one heater we only get ~4.5 FE/t through a turbine.
If someone can test this or give the optimal setup design, I will be very grateful. But I don't want use Resistive Heater to heat boiler (this is not efficient).

**Comment by bloodscon** [comment]
Main use of fuelwood heaters is for the brine towers also with meka turbines bigger is better the more blades = more power
i know in 1.12 using a heat gen with lava on/around it could produce quite a bit of heat

**Comment by emilekm2142** [comment]
So what is the solution? I have the same problem lol

**Comment by craft90** [comment]
I think it has been said before that the turbine is intended to work together with the nuclear generator to produce power. The boiler to turbine line has been useless since the nuclear generator was introduced. It's just not possible to use enough fuelwood generator to produce enough power to be worth the investment, unless the heat value is changed.
